api/src/main/webapp/WEB-INF/cron_test.yaml

```diff
@@ -1,7 +1,7 @@
 cron:
 - url: /v1/cron/checkObjectNameSize
   schedule: every 24 hours
-- url: /v1/cron/checkFreeTierBillingUsage
+- url: /v1/cron/checkForInitialCreditUsage
```

Comment:
this might break Monitoring alert https://github.com/all-of-us/workbench-terraform-modules/tree/main/modules/workbench/modules/monitoring/modules